src/popup/router/pages/Notifications.vue

```diff
@@ -3,7 +3,9 @@
     <ae-list class="noti-list">
       <ae-list-item fill="neutral" v-for="(noti, i) in notifications" :key="i" class="noti">
         <img src="../../../icons/icon_48.png" />
-        <span> {{ noti.content }} </span>
+        <span>
+          <router-link :to="noti.route"> {{ noti.content }} </router-link>
+        </span>
```

Comment:
I propose to don't use `router-link` if `noti.route` is empty:
```suggestion
        <Component
          :is="noti.route ? 'RouterLink' : 'span'"
          :to="noti.route"
        >
          {{ noti.content }}
        </span>
```

Reply:
This is breaking the build though.

Reply:
Should be ok now after https://github.com/aeternity/superhero-wallet/pull/77/commits/2430ccc04abf3dcd5855c4f83ba77751be35d309